org.freedesktop.Sdk.Extension.rust-stable.json

```diff
@@ -33,32 +33,32 @@
           "only-arches": [
             "i386"
           ],
-          "url": "https://static.rust-lang.org/dist/rust-1.27.2-i686-unknown-linux-gnu.tar.gz",
-          "sha256": "a34ef4b60acbab8df5d49ce9f168973cefe323059071a9ad2b32e901556491ed"
+          "url": "https://static.rust-lang.org/dist/rust-1.28.0-i686-unknown-linux-gnu.tar.gz",
+          "sha256": "de7cdb4e665e897ea9b10bf6fd545f900683296456d6a11d8510397bb330455f"
         },
         {
           "type": "archive",
           "only-arches": [
             "arm"
           ],
-          "url": "https://static.rust-lang.org/dist/rust-1.27.2-arm-unknown-linux-gnueabihf.tar.gz",
-          "sha256": "9948db9247f6d853a3c3392239ee08fc3ce1d2044cf282e2286403f4cbbcd799"
+          "url": "https://static.rust-lang.org/dist/rust-1.28.0-arm-unknown-linux-gnueabihf.tar.gz",
+          "sha256": "346558d14050853b87049e5e1fbfae0bf0360a2f7c57433c6985b1a879c349a2"
```

Comment:
This hash isn't correct, the correct one is: "5d93473fdaa85379ec83f5318b0316f968063198f9ef10ef2cadb5e73cfbc667"
take a look at this: https://static.rust-lang.org/dist/rust-1.28.0-arm-unknown-linux-gnueabihf.tar.gz.sha256

Reply:
whoops, write I will fix it in a sec